src/tools/BUILD

```diff
@@ -56,7 +56,17 @@ cc_binary(
     ],
     visibility = ["//visibility:public"],
     deps = [
-        "//external:api_manager_auth_lib",
+        "//src/api_manager/auth/lib",
+    ],
+)
+
+cc_binary(
+    name = "verify_token",
```

Comment:
where this binary is used? Add some comments?

Reply:
Done